server/src/main/kotlin/fi/oph/kitu/kotoutumiskoulutus/tehtavapankki/TehtavapankkiResponse.kt

```diff
@@ -0,0 +1,17 @@
+package fi.oph.kitu.kotoutumiskoulutus.tehtavapankki
+
+import com.fasterxml.jackson.annotation.JsonProperty
+
+data class TehtavapankkiResponse(
+    @JsonProperty("questionbanks")
+    val questionbanks: List<Questionbank>,
+) {
+    data class Questionbank(
```

Comment:
```suggestion
    data class QuestionBank(
```

Kirjoittaisin tuon englanniksi tekstissä kahtena sanana "question bank", joten pitäisi koodissakin. Toisaalta pitäisikö tämä olla suomeksi vaan koodissakin, eli `Kysymyspankki`?

Reply:
Mä ajattelin pitää siinä muodossa kuin missä se lähdejärjestelmässä on, välittämättä oikeasta muodosta.